src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -160,6 +160,22 @@ def mgmt_if_entry_table_state_db(if_name):
 
     return b'MGMT_PORT_TABLE|' + if_name
 
+def get_sai_id_key(namespace, sai_id):
+    sai_id = sai_id.decode()
```

Comment:
>decode [](start = 20, length = 6)

decode and encode will applied to single ASIC and this could be optimized.

Reply:
Fixed.